src/main/scala/pipelines/images/imagenet/ImageNetSiftLcsFV.scala

```diff
@@ -48,7 +48,7 @@ object ImageNetSiftLcsFV extends Serializable with Logging {
         new BatchPCATransformer(convert(csvread(new File(fname)), Float).t)
       case None =>
         val pca = sampledColumns andThen
-            (ColumnPCAEstimator(numPCADesc), trainingData)
```

Comment:
uh not particularly, I just haven't tested these w/ the optimized nodes yet and am trying to minimize risks. I guess the integration tests will try this out for us?